azure-pipelines.yml

```diff
@@ -39,11 +39,73 @@ jobs:
       displayName: "Run tests"
       workingDirectory: $(Build.BinariesDirectory)/build
 
+    - script: |
+        echo "##vso[task.setvariable variable=Status;isOutput=true]1"
+      name: JobResult
+
+
+  - job: LinuxDebug
+
+    pool:
+      vmImage: 'Ubuntu-16.04'
+
+    container: trailofbits/osql-experimental:ubuntu-18.04
+
+    steps:
+    - script: mkdir $(Build.BinariesDirectory)/build
+      displayName: "Create build folder"
+
+    - task: CMake@1
+      displayName: "Configure osquery"
+      inputs:
+        workingDirectory: $(Build.BinariesDirectory)/build
+        cmakeArgs: -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_TESTING=ON -DOSQUERY_NO_DEBUG_SYMBOLS=ON $(Build.SourcesDirectory)
```

Comment:
Can these arguments be listed per-line?

Reply:
I tried and it seems to work, I'll update the commit.